python/vowpalwabbit/pyvw.py

```diff
@@ -540,7 +540,7 @@ def __init__(self, label, prediction=None):
             self.prediction = prediction
 
     def from_example(self, ex):
-        self.prediction = ex.get_multiclass_probabilities()
```

Comment:
Instead of changing this call, I think we should add `get_multiclass_probabilities()` to the attributes. After this line https://github.com/VowpalWabbit/vowpal_wabbit/blob/master/python/pylibvw.cc#L870

@jackgerrits thoughts?

Reply:
Multiclass probs isn't a prediction type in example.h: https://github.com/VowpalWabbit/vowpal_wabbit/blob/2d00e4514f2ca72fb64b18075100d62c81395e73/vowpalwabbit/example.h#L43

But for some reason in python multiclass probs is it's own prediction type. I don't know how that came to be. But you can see here that it maps to get_scalars:
https://github.com/VowpalWabbit/vowpal_wabbit/blob/2d00e4514f2ca72fb64b18075100d62c81395e73/python/vowpalwabbit/pyvw.py#L72

We should probably use the get_prediction function with the type pMULTICLASSPROBS. That way intention is clearer and the conversion logic is all in one place.

Reply:
As, it points to `ec.get_scalars()`, we could directly call it using get_scalars, instead of calling `get_prediction function` to avoid unnecessary function call. Is it a good idea?

Reply:
I think we want to opt for using get_prediction here for maintainability. (the python bindings are likely to be refactored soonish as well)

Reply:
Okay, sure. I will change that.